src/js/experiments/snippetFeedback.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * The SnippetFeedback class is responsible for dynamically adding a user
+ * feedback form whenever a user clicks on the "copy code" button.
+ *
+ * This class is instantiated for each snippet in a page.
+ * Most of the methods are for dynamically adding dom elements for
+ * capturing user feedback.
+ *
+ * */
+export class SnippetFeedback {
+  constructor(snippetElement) {
+    // Dom elements
+    this.snippetElement = snippetElement;
+    this.codeSnippetContainer = null;
+    this.wasThisHelpfulContainer = null;
+    this.feedbackForm = null;
+    this.maxCharCount = 240;
```

Comment:
properties like this could easily be part of an optional properties/options argument with default fallback values if nothing is specified.

Also it's preferable for constants to be all uppercase.

https://softwareengineering.stackexchange.com/questions/319688/what-is-the-history-for-naming-constants-in-all-uppercase

Reply:
Done